test/RewardManager-test.js

```diff
@@ -1260,7 +1261,7 @@ contract("RewardManager", (accounts) => {
       );
     });
 
-    it("can transfer reward safe", async () => {
+    it.only("can transfer reward safe", async () => {
```

Comment:
test still isolated, that'll be what's causing the coverage failure

Reply:
Thx!